third_party/3/bleach/linkifier.pyi

```diff
@@ -0,0 +1,30 @@
+from typing import Any, List, MutableMapping, Protocol, Pattern, Iterable, Optional
+
+_Attrs = MutableMapping[Any, str]
+
+class _Callback(Protocol):
+    def __call__(self, attrs: _Attrs, new: bool = ...) -> _Attrs: ...
+
+DEFAULT_CALLBACKS: List[_Callback]
+
+TLDS: List[str]
```

Comment:
Text actually (the file has unicode_literals). I think most of the strs below should also be Text.